doc/api/live_events.md

```diff
@@ -286,3 +286,45 @@ by `asset_type` and `asset_id`.
 | `body` | The content of the submission, if it was submitted directly in a text field. (possibly truncated) |
 | `url` | The URL of the submission (for 'online_url' submissions) |
 | `attempt` | This is the submission attempt number.|
+
+#### `attachment_created`
+
+| Field | Description |
+| ----- | ----------- |
+| `user_id` | The Canvas id of the user associated with the attachment. |
+| `attachment_id` | The Canvas id of the new attachment. |
+| `display_name` | The display name of the attachment (possibly truncated). |
+| `filename` | The file name of the attachment (possibly truncated). |
+| `hidden` | Whether the file is flagged as hidden |
+| `unlock_at` | The unlock date (attachment is unlocked after this date) |
+| `lock_at` | The lock date (attachment is locked after this date) |
+| `updated_at` | The time at which this attachment was last modified in any way |
+
+#### `attachment_updated`
+
+`attachment_updated` events are triggered when an attachment's `display_name` is updated.
+
+| Field | Description |
+| ----- | ----------- |
+| `user_id` | The Canvas id of the user associated with the attachment. |
+| `attachment_id` | The Canvas id of the new attachment. |
+| `display_name` | The display name of the attachment (possibly truncated). |
+| `old_display_name` | The old display name of the attachment (possibly truncated). |
```

Comment:
`old` is fine for the variable name, but they appear to be using `previous` in the description.